test/projects-summary.spec.ts

```diff
@@ -313,6 +366,24 @@ describe('Projects Summary Panel', () => {
     expect(showProjectsCallCount).toBe(1);
   });
 
+  it('should call the start guided tour callback when the start tour button is clicked', () => {
+    createProjectSummary();
+
+    var element = componentTest.rawElement;
+
+    $timeout.flush();
+
+    var starTourButton = jQuery(element).find('.getting-started-button ');
+    expect(starTourButton.length).toBe(1);
+
+    expect(showTourCount).toBe(0);
+
+    componentTest.eventFire(starTourButton[0], 'click');
+    componentTest.scope.$digest();
+
+    expect(showTourCount).toBe(1);
+  });
+
   it('should call the select project callback when the project name is clicked', () => {
     createProjectSummary();
```

Comment:
Hi, I noticed in your unit test:


```
it('should hide the create button when not allowed', () => {
    expectedCanCreate = false;
    createProjectSummary();

    var element = componentTest.rawElement;

    $timeout.flush();

    var createButton = jQuery(element).find('.create-button');
    expect(createButton.length).toBe(1);
  });
```
Shouldn't that last expect be '...toBe(0)' when the create button is hidden?